app/admin-tab/auth/github/controller.js

```diff
@@ -99,16 +101,25 @@ export default Ember.Controller.extend({
         'allowedIdentities' : [],
       });
 
-      // Send authenticate immediately so that the popup isn't blocked,
-      // even though the config isn't necessarily saved yet...
       this.get('github').setProperties({
         hostname : githubConfig.get('hostname'),
         scheme   : githubConfig.get('scheme'),
         clientId : githubConfig.get('clientId')
       });
-      this.send('authenticate');
```

Comment:
This is probably going to break with normal pop-up blockers as the comment said...